src/yang/drned-xmnr.yang

```diff
@@ -275,9 +275,19 @@ module drned-xmnr {
             leaf target-format {
               type state-file-format;
             }
-            leaf overwrite {
-              type boolean;
-              default false;
+            choice conflicts {
+              default overwrite;
+              case overwrite {
+                leaf overwrite {
+                  type boolean;
+                  default false;
+                }
+              }
+              case skip-existing {
+                leaf skip-existing {
+                  type empty;
+                }
+              }
```

Comment:
Maybe I misunderstand the code but didn't we say that "continue where I left off" should be the default?

Reply:
The current behavior keeps "old" behavior as default for legacy/compatibility reasons.
This can be changed easily by changing default case statement before merging in PR, question is which approach is safer, see e.g. latest comments in #66 discussion...